### PR TITLE
test(blue-sdk): add tests for borrowable assets

### PR DESCRIPTION
**Pull request by Rubilmax**
Fixes INTER-1458

There was no issue in the end

### CODE REVIEW

File: packages/blue-sdk/test/Market.test.ts [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    await morpho.borrow(
+      config,
+      market.getMaxBorrowableAssets(position),
+      0n,
+      signer.address,
+      signer.address,
+    );
+  });
+
+  it("should borrow borrowable assets in a galaxy far far away", async () => {
```
Comment (oumar-fall) [inline packages/blue-sdk/test/Market.test.ts:87]: ??? 🤣

File: packages/blue-sdk/test/Market.test.ts
Comment (oumar-fall) [inline packages/blue-sdk/test/Market.test.ts:1]: There's no check in the test, is it normal?

File: packages/blue-sdk/test/Market.test.ts [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+import { ChainId, Market, MarketConfig, MathLib, addresses } from "../src";
+
+describe("Market", () => {
+  let signer: SignerWithAddress;
+
+  setUp(async () => {
+    signer = (await ethers.getSigners())[0]!;
+  });
+
+  it("should borrow borrowable assets", async () => {
```
Comment (oumar-fall) [inline packages/blue-sdk/test/Market.test.ts:27]: Maybe you should check that borrowing a bit more than `maxBorrowableAssets` reverts?

File: packages/blue-sdk/test/Market.test.ts [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+      signer.address,
+    );
+  });
+
+  it("should borrow borrowable assets in a galaxy far far away", async () => {
+    const { morpho: morphoAddress } = addresses[ChainId.EthMainnet];
+    const morpho = MorphoBlue__factory.connect(morphoAddress, signer);
+
+    const config = new MarketConfig({
+      loanToken: "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
```
Comment (oumar-fall) [inline packages/blue-sdk/test/Market.test.ts:92]:
```suggestion
      loanToken: "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48", // USDC
```

Comment (oumar-fall) [reply packages/blue-sdk/test/Market.test.ts:92]: specifying which tokens it is makes it easier for reviewing (esp. understanding the following numbers of decimals)

File: packages/blue-sdk/test/Market.test.ts [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+    });
+
+    const collateral = MathLib.WAD * 10_000_000_000n;
+    await deal(config.collateralToken, signer, collateral);
+    await ERC20__factory.connect(config.collateralToken, signer).approve(
+      morphoAddress,
+      collateral,
+    );
+    await morpho.supplyCollateral(config, collateral, signer.address, "0x");
+    await morpho.borrow(config, 1_000000n, 0n, signer.address, signer.address);
```
Comment (oumar-fall) [inline packages/blue-sdk/test/Market.test.ts:106]:
```suggestion
    await morpho.borrow(config, parseUnits("1", 6), 0n, signer.address, signer.address);
```

File: packages/blue-sdk/test/Market.test.ts (hunk 1 above)
Comment (oumar-fall) [reply packages/blue-sdk/test/Market.test.ts:87]:
```suggestion
  it("should borrow borrowable assets in a far far future", async () => {
```

Comment (oumar-fall) [reply packages/blue-sdk/test/Market.test.ts:87]:
```suggestion
  it("should borrow borrowable assets in n extreme future", async () => {
```

Comment (oumar-fall) [reply packages/blue-sdk/test/Market.test.ts:87]: More boring but more accurate 🤣

**Review by oumar-fall — DISMISSED** [review]

File: packages/blue-sdk/test/Market.test.ts
Comment (Rubilmax) [reply packages/blue-sdk/test/Market.test.ts:1]: Having the transaction not revert is a test in itself
But I'll add the expect revert anyway

Comment (Rubilmax) [reply packages/blue-sdk/test/Market.test.ts:1]: I added the test that it reverts but not in an extreme future where it is quite unprecise

**Pull request merged**